README.rst

```diff
@@ -4,10 +4,117 @@ Overview
 
 .. image:: https://secure.travis-ci.org/quora/semidbm2.png?branch=master
    :target: http://travis-ci.org/quora/semidbm2
+	    
+.. image:: https://img.shields.io/pypi/v/semidbm2.svg
+    :target: https://pypi.python.org/pypi/semidbm2/
+    :alt: Latest Version
 
+.. image:: https://img.shields.io/pypi/pyversions/semidbm2.svg
+    :target: https://pypi.python.org/pypi/semidbm2/
+    :alt: Supported Python versions
+
+.. image:: https://img.shields.io/pypi/implementation/semidbm2.svg
+    :target: https://pypi.python.org/pypi/semidbm2/
+    :alt: Supported Python implementations
+
+.. image:: https://img.shields.io/pypi/l/semidbm2.svg
+    :target: https://pypi.python.org/pypi/semidbm2/
+    :alt: License
+
+.. image:: https://img.shields.io/pypi/wheel/semidbm2.svg
+    :target: https://pypi.python.org/pypi/semidbm2/
+    :alt: Wheel Status
+
+Semidbm2 is an up-to-date fork of [Semidbm](https://github.com/jamesls/semidbm).
 
 Semidbm is a fast, pure python implementation of a dbm, which is a
-persistent key value store. This is a fork of https://github.com/jamesls/semidbm
-which includes a fix for read-only mode. In the original fork, read only mode
-will still attempt to create the data directory if it does not exist, may write
-headers, and uses read and write flags.
+persistent key value store. It allows you to get and set keys through
+a dict interface::
+
+    import semidbm
+    db = semidbm.open('testdb', 'c')
+    db['foo'] = 'bar'
+    print db['foo']
+    db.close()
+
+These values are persisted to disk, and you can later retrieve
+these key/value pairs::
+
+    # Then at a later time:
+    db = semidbm.open('testdb', 'r')
+    # prints "bar"
+    print db['foo']
+
+
+It was written with these things in mind:
+
+* Pure python, supporting python 2.6, 2.7, 3.3, and 3.4.
+* Cross platform, works on Windows, Linux, Mac OS X.
+* Supports CPython, pypy, and jython (versions 2.7-b3 and higher).
+* Simple and Fast (See `Benchmarking Semidbm <http://semidbm.readthedocs.org/en/latest/benchmarks.html>`__).
+
+
+Supported Python Versions
+=========================
+
+Semidbm supports python 2.6, 2.7, 3.3, and 3.4.
```

Comment:
Will you update this later? 2.6, 3.3, and 3.4 are dead, so we probably shouldn't spend more effort supporting them.

Reply:
Ah yeah thanks, we already dropped support for 2.6 & 3.3, this should be updated